aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1,52 +1,28 @@
-
-Many definitions for tone and voice exist, however, reading through the
-various definitions, a common picture emerges:
-
-| Voice                         | Tone                                        |
-| ----------------------------- | ------------------------------------------- |
-| Who they hear                 | What they hear                              |
-| Personality                   | Attitude                                    |
-| How you are doing the writing | Who they hear                               |
-| Style                         | How your message comes across               |
-| What makes us uniquely us     | How we communicate to our specific audience |
-
-### Voice
-
-Voice is who the reader "hears" delivering messaging; embedded within it is the
-worldview of the author. Voice can be spoken, witty, friendly, formal, etc.
-At HPE, the desire is for HPE's voice to be perceived as intelligent, experienced,
-objective, inclusive, and trustworthy &mdash; supporting HPE's worldview which
-believes in accelerating what's next, making bold moves, the power of _yes we can_, and
-being a force for good.
-
-A person's voice typically does not change once established. If HPE's voice is
-inconsistent, people will notice. For example, people notice the difference
-between writing academically when compared to sounding like a 5-year old. This
-kind of inconsistency can result in lack of trust.
-
-### Tone
-
-Tone has to do with what is conveyed or how it comes across. It can express seriousness, urgency, playfulness, objectivity, intimacy, condescension, etc.
-
-Unlike voice, tone can change based on the target audience, context, and subject
-matter. It is situational. For example, tone is very different when giving
-directions during an emergency compared to when ordering a pizza.
-
-Because tone is situational, it may be tweaked to appropriately fit the context
-within an HPE application. While tone may change, the message should be consistently
-delivered in [HPE's voice](#voice) supporting the perception of intelligence,
-experience, objectivity, and inclusivity.
+The HPE Design System employs text-based content that cultivates an exceptional user experience.
+<!-- The above sentence will be the leading line and moved to the Voice and Tone card in Foundations once approved -->
+
+Every word we write is a carefully-considered concept that voices our values: 
+
+- Open: We foster inclusive experiences that are personal, accessible, and transparent.
+- Courageous: We empower users with decisive and experienced guidance.
+- Inspired: We share our drive of 'yes we can' with users.
+
+Refer to the guidance for [HPE's Voice](https://brandcentral.hpe.com/brand-central/content/voice) for more information.
+
+## Using Our Voice
+
+Our voice is intrinsic to our content which is fundamental to the user experience. We exercise our voice through situational tones that speak to each context, audience, and subject matter. 
```

Comment:
This sentence structure is a little confusing to me. Is "which is fundamental to the user experience" referring to content or voice? Potentially could rephrase to make it a little smoother.